src/main/java/org/mockito/internal/progress/MockingProgressImpl.java

```diff
@@ -143,4 +146,18 @@ public void setVerificationStrategy(VerificationStrategy strategy) {
     public VerificationMode maybeVerifyLazily(VerificationMode mode) {
         return this.verificationStrategy.maybeVerifyLazily(mode);
     }
-}
+
+    public void setStubbingListener(StubbingListener stubbingListener) {
+        this.stubbingListener = stubbingListener;
+    }
+
+    public StubbingListener getStubbingListener() {
+        if (this.stubbingListener == null) {
+            return new StubbingListener() {
```

Comment:
great idea. Single listener is simpler, so I would go ahead and merge it as is. However, I think a list of listeners makes much more sense. So I would just do it sometime before 2.0 :)